FaceID/faceID.py

```diff
@@ -285,35 +282,42 @@ def searchFace():
 
         count = 0
         accuracyList = []
+        avgAccuracy = -1
+        accThreshold = 25
+        lock = True
+
+        camWait = 10 # waits this amount of time in seconds to detect a face
+        timeStamp = time.time()
+
+        while (np.abs(int((time.time() - timeStamp))) < camWait):
 
-        while True:
-            
             ret, img = cam.read()
             gray = cv2.cvtColor(img,cv2.COLOR_BGR2GRAY)
-            
-            faces = faceCascade.detectMultiScale( 
+
+            faces = faceCascade.detectMultiScale(
                 gray,
                 scaleFactor = 1.2,
                 minNeighbors = 5,
                 minSize = (int(minW), int(minH)),
             )
 
+
             for(x,y,w,h) in faces: # detects face
 
+                timeStamp = time.time()
+
                 EmpID, accuracy = recognizer.predict(gray[y:y+h,x:x+w]) # recognizes face
-                
+
                 accuracy = round(100 - accuracy)
-                    
-                if (0 < accuracy <= 100):
+
+                if (accuracy <= 100) and (accuracy > 0):
```

Comment:
This can be simplified to: `if (0 < accuracy <= 100):`

Reply:
Fixed!